lib/nostrum/bot.ex

```diff
@@ -268,14 +272,18 @@ defmodule Nostrum.Bot do
   @spec fetch_all_bots() :: [%{name: name(), pid: pid(), bot_options: bot_options()}]
   def fetch_all_bots do
     Registry.select(Nostrum.Bot.Registry, [
-      {{:"$1", :"$2", :"$3"}, [], [%{name: :"$1", pid: :"$2", bot_options: :"$3"}]}
+      {
+        {{:nostrum_bot, :"$1"}, :"$2", :"$3"},
+        [],
+        [%{name: :"$1", pid: :"$2", bot_options: :"$3"}]
+      }
     ])
   end
 
   @doc false
   @spec fetch_bot_pid(name()) :: pid() | nil
   def fetch_bot_pid(name) do
-    case Registry.lookup(Nostrum.Bot.Registry, name) do
+    case Registry.lookup(Nostrum.Bot.Registry, {:nostrum_bot, name}) do
```

Comment:
Can you make this change on lines 329 and 352 too

Reply:
Done